src/mono/wasm/wasm.proj

```diff
@@ -98,22 +97,21 @@
     </GetFileHash>
     <ItemGroup>
       <_WasmBundleTimezonesWithHashes Update="@(_WasmBundleTimezonesWithHashes)">
-        <DestinationFile>$(WasmObjDir)\wasm-bundled-$([System.String]::Copy(%(_WasmBundleTimezonesWithHashes.FileHash))).c</DestinationFile>
```

Comment:
Shouldn't this look similar to wasi.proj, not using GetFileHash since EmitBundleSourceFiles take care of content hashing and return the correct paths as part of its output item?

Reply:
Right, I mainly just removed the DestinationFile, but also updated to not use the GetFileHash task, thanks!